### ISSUE TITLE
Start Pos Switcher Spawn Point Bug

### ISSUE THREAD

**Issue by BrownieGamingX5** [absoIute/Mega-Hack-Pro-Future#547]
Sometimes using Start Pos Switcher can make you spawn in a completely random spot. I don't know if this has been mentioned prior by the way.

Again, here is a video of showcasing this, due to GitHub's file size limit: https://www.youtube.com/watch?v=ChITGjF9uik

**Comment by absoIute** [comment]
If you enable StartPos Bug Fix does it help?

**Comment by BrownieGamingX5** [comment]
No. It doesn't even help at all. I have it enabled, and it doesn't do anything to fix the bug.

**Comment by Nicko2102** [comment]
Just a thought, it might be an issue with mirror portals
